package.json

```diff
@@ -106,7 +106,8 @@
     "rimraf": "^2.6.1",
     "semver": "^5.1.0",
     "shelljs": "^0.7.7",
-    "shx": "^0.2.2"
+    "shx": "^0.2.2",
+    "react-native": "0.57.7"
```

Comment:
I need to add react-native to `devDependecies` in order to be able to buid `/react-native/android` independently. I don't assume this a problem? @kneth ?

Reply:
It means that you will have to install React Native even when you only develop for node.js. But I think it is acceptable.